requirements-build.txt

```diff
@@ -1 +1,2 @@
-cmake~=3.30
\ No newline at end of file
+cmake~=3.23
+pybind11-stubgen==2.5.1
```

Comment:
looks like we don't need this file anymore, since all those requirements will be taken from toml

Reply:
it's used by Jenkins